ApollosProject/apollos-prototype
# Don't cache the user exists query

## DESCRIPTION

### What does this PR do, or why is it needed?

1. Caching this field doesn't save us much
2. There's a chance that calling this field again (with a cached value) could return the wrong value and harm the user experience. 
3. It makes testing some things hard. 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

You can run a query that tests this field and check the HTTP headers returned (there won't be any cache headers if this PR is functioning)

### What automated tests did you write?

n/a

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
